gravatar-quickeditor/src/main/res/values/strings.xml

```diff
@@ -1,40 +1,40 @@
+    <string name="gravatar_avatar_picker_title">Avatars</string>
+    <string name="gravatar_avatar_picker_title_empty_state">Let’s setup your avatar</string>
+    <string name="gravatar_avatar_picker_description">Choose or upload your favorite avatar images and connect them to your email address.</string>
+    <string name="gravatar_avatar_selected_confirmation">Avatar updated! May take a few minutes to appear everywhere.</string>
+    <string name="gravatar_avatar_upload_error_action">Retry</string>
+    <string name="gravatar_avatar_picker_upload_image">Upload image</string>
+    <string name="gravatar_avatar_picker_choose_a_photo">Choose a Photo</string>
+    <string name="gravatar_avatar_picker_take_photo">Take Photo</string>
+    <string name="gravatar_photo_library_icon_description">Photo library</string>
+    <string name="gravatar_capture_photo_icon_description">Capture Photo</string>
+    <string name="gravatar_avatar_selection_error">Ooops, there was an error selecting the avatar.</string>
+    <string name="gravatar_avatar_picker_server_error_title">Ooops</string>
+    <string name="gravatar_avatar_picker_server_error_message">Something went wrong and we couldn\'t connect to Gravatar servers.</string>
+    <string name="gravatar_avatar_picker_unknown_error_message">Something went terribly wrong.</string>
+    <string name="gravatar_avatar_picker_error_retry_cta">Try again</string>
+    <string name="gravatar_avatar_picker_session_error_title">Session expired</string>
+    <string name="gravatar_avatar_picker_session_error_message">Session expired for security reasons. Please log in to update your Avatar.</string>
+    <string name="gravatar_avatar_picker_session_error_cta">Log in</string>
+    <string name="gravatar_avatar_picker_session_error_no_login_message">Session expired for security reasons.</string>
+    <string name="gravatar_avatar_picker_session_error_close_cta">Close</string>
+    <string name="gravatar_avatar_picker_network_error_title">No internet</string>
+    <string name="gravatar_avatar_picker_network_error_message">Uh oh! Looks like you\'re not connected to the internet.</string>
+    <string name="gravatar_login_required">Login required</string>
+    <string name="gravatar_login_required_message">To modify your Gravatar profile you need to log&#160;in first</string>
+    <string name="gravatar_oauth_wrong_email_authenticated_error_message">It looks like you used the wrong email to log in. Please try again using %s this time. Thanks!</string>
+    <string name="gravatar_happy_face_image_content_description">Happy face image</string>
+    <string name="gravatar_upload_image_content_description">Upload image</string>
+    <string name="gravatar_failed_to_load_avatar_content_description">Upload failed, tap to see actions.</string>
+    <string name="gravatar_avatar_upload_failure_dialog_title">Couldn\'t upload image</string>
+    <string name="gravatar_avatar_upload_failure_dialog_remove_upload">Remove upload</string>
```

Comment:
I've started using `gravatar_qe` for the quick editor module (you can see below). I thought it would be safer to use different prefixes per module. WDYT?

Reply:
Sounds good 👍

Reply:
Done!